Makefile

```diff
@@ -2,7 +2,7 @@ CC ?= gcc
 AR ?= ar
 RANLIB ?= ranlib
 
-CCFLAGS ?= -Wall -Wextra -Wshadow -Wconversion -std=c89 -pedantic -Wno-declaration-after-statement -O2
+CCFLAGS ?= -Wall -Wextra -Wshadow -Wconversion -std=c99 -pedantic -O2 -g
```

Comment:
btw, we can't claim to use ANSI C from now on, but I personally completely support this

Reply:
Sure, I'll change the documentation. I don't think anyone will care that it's C99.

I do a lot of embedded work where C89 is sometimes required, so it was mostly out of habit that I called this ANSI, rather than a careful decision.